scripts/run-db

```diff
@@ -53,7 +51,7 @@ COMMAND=$1
 remove_database() {
   if [ "$clean_db" = "true" ]; then
     log_info "erasing existing database state"
-    rm -r "$TMP_DIR"/postgres-data
+    rm -r "$ROOT_DIR"/build/tmp/postgres-data
```

Comment:
This is the actual location of the test database, see https://github.com/DataBiosphere/jade-data-repo/blob/develop/scripts/compose.yaml#L36